src/common/upload.ts

```diff
@@ -37,6 +40,11 @@ import {
 import { TurboHTTPService } from './http.js';
 import { TurboWinstonLogger } from './logger.js';
 
+export const createDelegatedPaymentApprovalTagName = 'x-approve-payment';
+export const approvalAmountTagName = 'x-amount';
+export const approvalExpiresBySecondsTagName = 'x-expires-seconds';
+export const revokeDelegatePaymentApprovalTagName = 'x-delete-payment-approval';
```

Comment:
nit: this could be a map to avoid redundant variable names

Reply:
[d7a6f5e](https://github.com/ardriveapp/turbo-sdk/pull/196/commits/d7a6f5e8a8aebc717cab30d43c982c04ef422e5b)